modules/REST/V1/Options.php

```diff
@@ -0,0 +1,218 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\REST\V1;
+
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+use WP_Syntex\Polylang\Options\Options as Options_Registry;
+use WP_Syntex\Polylang\REST\Abstract_Controller;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Options REST controller.
+ *
+ * @since 3.7
+ */
+class Options extends Abstract_Controller {
+	/**
+	 * @var Options_Registry
+	 */
+	private $options;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 3.7
+	 *
+	 * @param Options_Registry $options Options registry.
+	 */
+	public function __construct( Options_Registry $options ) {
+		parent::__construct();
+		$this->rest_base = 'options';
```

Comment:
I thnk about it only now, but should we use the same term as WordPress does? i.e. `settings`?

Reply:
It's a good idea I think, so it's consistent.

Reply:
Done in [fb165da](https://github.com/polylang/polylang/pull/1569/commits/fb165da750a8c4703a47bf3cd47eeb6b3df15e49).